GettingStarted/ibmdbpy_GettingStarted_2-geo_basics.ipynb

```diff
@@ -0,0 +1,1621 @@
+    "idadb=IdaDataBase(dsn=connection_string)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Congratulations! You successfully connected to Db2 with ibmdbpy! When you are done, use `idadb.close()` to close the connection. To reconnect, or if the connection was broken, just use `idadb.reconnect()`. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__Verbosity and autocommit__\n",
+    "\n",
+    "The verbose mode automatically prints all SQL-communication between ibmdbpy and Db2, which can be very useful for debugging or understanding how ibmdbpy works. Choose the mode with `set_verbose()` or by setting the `verbosity` option when defining the IdaDataBase object. We encourage you to take a look at the prints in the first place, then feel free to silence the verbose. Note that printing adds delay when running cells.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Verbosity\n",
+    "from ibmdbpy.utils import set_verbose\n",
+    "set_verbose(False) # set it to True if you want to see the detail of ibmdbpy operations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By default the environment variable `AUTOCOMMIT` is then set to True, which means that every SQL statement which is submitted through the connection is executed within its own transaction and then committed implicitly. When you close the connection to Db2, if the environment variable `AUTOCOMMIT` is set to False, then all changes after the last explicit commit are discarded. \n",
+    "\n",
+    "Let's get to it!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Open and define IdaGeoDataFrames\n",
+    "\n",
+    "Let's explore the functionalities of ibmdbpy's geospatial extension! In this notebook, we will get familiar with IdaGeoDataFrames. An IdaGeoDataFrame is a reference to a spatial table in a remote instance of Db2. It has inherited the properties of an IdaDataFrame and benefits from additional functionalities derived from Db2 spatial extension. \n",
+    "\n",
+    "In the examples below, we simply use sample data available out of the box in Db2 Warehouse. The`SAMPLES.GEO_COUNTY` dataset contains geographical and administrative data about US counties."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "__Load the data__"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* Method 1: open the data directly as IdaGeoDataFrame\n",
+    "\n",
+    "Simply call the IdaGeoDataFrame class. The first argument is the name of the IdaDataBase object which holds the connection to the database, the second argument designates the Db2 table you want to open. Optionally you may set an `indexer` column and a `geometry` column.\n",
```

Comment:
Usually, only functions/methods, but not classes are called. 
```suggestion
    "Simply create an object of the IdaGeoDataFrame class. The first argument is the name of the IdaDataBase object which holds the connection to the database, the second argument designates the Db2 table you want to open. Optionally you may set an `indexer` column and a `geometry` column.\n",
```

Reply:
changed